chore: remove usage of brand color namespace in docs/types/tests (#4916)
aws-amplify/amplify-ui: .changeset/small-masks-fail.md

---
"@aws-amplify/ui-react": patch
---

> chore: Removes the brand namespace from the BrandColor type and updates usage across docs/examples.

Author: ```suggestion
chore: Removes the brand namespace from the BrandColorKeys type and updates usage across docs/examples.
```

> chore: Removes the brand namespace from the `BrandColorKeys` type, splits it into `PrimaryColorKeys` and `SecondaryColorKeys`, and updates usage across docs/examples.

Author: ```suggestion
chore: Removes the brand namespace from the `BrandColorKeys` type, splits it into `PrimaryColorKey` and `SecondaryColorKey`, and updates usage across docs/examples.
```

---

> "@aws-amplify/ui-react-native": patch

Reviewer: ```suggestion
```
